etc/kayobe/environments/stackhpc-baremetal/inventory/group_vars/baremetal-redfish/ironic

```diff
@@ -0,0 +1,19 @@
+---
+
+ironic_driver: redfish
+
+ironic_driver_info:
+  redfish_system_id: "{{ ironic_redfish_system_id }}"
+  redfish_address: "http://192.168.33.3:34343"
+  redfish_username: "{{ ironic_redfish_username }}"
+  redfish_password: "{{ ironic_redfish_password }}"
+  redfish_verify_ca: "{{ ironic_redfish_verify_ca }}"
+
+ironic_redfish_verify_ca: false
+ironic_redfish_address: "{{ redfish_address }}"
+ironic_redfish_system_id:
+ironic_redfish_username: "{{ secrets_redfish_baremetal_username | default('') }}"
+ironic_redfish_password: "{{ secrets_redfish_baremetal_password | default('') }}"
+ironic_resource_class: "example_resource_class"
```

Comment:
Lets remove this default I think, lets do "| hit('this must be defined')" instead?